src/components/Popover/README.md

```diff
@@ -162,6 +162,28 @@ automatically, including smart position updates to ensure Popover visibility,
 we recommend to involve an external library designed specifically for this
 purpose.
 
+To position the popover, you need to provide the `placementStyle` prop with the
+style you want to apply to the popover. This prop is filtered and should only
```

Comment:
I think we should reword:

> This prop is filtered and should only …

to:

> This prop should only …

The code does not filter anything. It merely shows warning and error when in dev mode.  I find this sufficient, but the docs should match the code.